android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -765,11 +804,21 @@ public void receiveCommand(WebView root, int commandId, @Nullable ReadableArray
   }
 
   @Override
-  public void onDropViewInstance(WebView webView) {
-    super.onDropViewInstance(webView);
-    ((ThemedReactContext) webView.getContext()).removeLifecycleEventListener((RNCWebView) webView);
-    ((RNCWebView) webView).cleanupCallbacksAndDestroy();
-    mWebChromeClient = null;
+  public void onDropViewInstance(RNCWebView wrapper) {
+    super.onDropViewInstance(wrapper);
+
+    // View has already been reattached, do nothing
```

Comment:
nit: Can we clarify that it's the internal webview (not the RNCWebView) that has been reattached in the code comment?

Reply:
done